conf/ocsci/vsphere_upi_vars.yaml.example

```diff
@@ -0,0 +1,27 @@
+---
+ENV_DATA:
+  platform: 'vsphere'
+  deployment_type: 'upi'
+  # aws region
+  region: 'us-east-2'
+  base_domain: 'qe.rh-ocs.com'
+  # vsphere details
+  vsphere_server: 'vcenter-ocsqe.scl.lab.tlv.redhat.com'
+  vsphere_user: '<user>'
+  vsphere_password: '<pwd>'
+  vsphere_cluster: '<cluster>'
+  vsphere_datacenter: '<data center>'
+  vsphere_datastore: '<data store>'
+  machine_cidr: '<machine_cidr>'
+  ipam: '139.178.89.254'
+  ipam_token: '<token>'
+  # use vm_network if you are using separate VLAN
+  vm_network: 'VLAN_172'
+  vm_template: 'rhcos-410.8.20190520.1'
+  gateway: '<gateway>'
+  dns: '<dns>'
+  httpd_server: 'server_name'
+  path_to_upload: '/var/www/html/ocs4qe'
+  user: 'ocs4qe' #password-less ssh is needed to server
```

Comment:
For this password-less ssh we we need to use private key. So it means that user needs to use ssh-copy-id commant co copy there their own public key as a prerequisites. Is it somewhere described? For jenkins machines I've just copied libra pub key there.

Reply:
Basically we need to upload bootstrap.ign to some location over internet where we can access. In this PR I am using httpd server to upload bootstrap ignition. ( thats the reason using passwordless ssh )

Reply:
As we discussed on hangout chat I think we can define password here in this file to make it easier.
So add here the httpd_server_password and change the user to httpd_server_user and update in skeleton as well

Reply:
Done